docs/product-editor-development/product-editor-extensibility-guidelines.md

```diff
@@ -52,31 +50,9 @@ Field extensions are the simplest type of extension. They can add new, replace,
 * Entering custom data, f.e. color or date and time
 * Selecting from a third-party system, f.e. warranty type
 
-### Group 
-
-Groups add extra fields to existing form sections. They are small forms with a low to medium level of complexity. This interface location works best for extensions that add extra features that build off an existing Woo functionality.
-
-#### ✅ What they are for
-
-* Relevant features that can be crucial to merchants’ product creation flow
-* 2-5 field forms with limited functionality, f.e. dimensions or tax settings
-* Lists of items, f.e. attachments, channels, or accounts
-
-#### ❌ What they aren’t for
-
-* Simple extensions with 1-2 fields
-* Multi-step forms and complex tables
-* Read-only descriptions, setup guides, and advertisements
-
-#### Use cases
-
-* Adding extra product details, f.e. measurements under Shipping
-* Setting up social channels in the Visibility section in the General tab
-* Changing the VAT tax settings in the Pricing tab
-
-### Section
+### Sections
```

Comment:
Although from an implementation standpoint, sections and sub-sections are identical, we should have some guidance on when to create a section and when to create a sub-section (what was originally called a "group" in the draft guidelines).